index/templates/home/settings.html

```diff
@@ -20,59 +20,27 @@
     <section class="container site-content animate__animated animate__backInDown animate__faster">
         <h1>Settings</h1>
     </section>
-    <div class="container site-content animate__animated animate__fadeIn animate__faster table-responsive">
+    <div class="container site-content animate__animated animate__fadeIn animate__faster table-responsive" style="font-family: 'arcadeclassicregular', sans-serif">
         <table class="table text-white" style="width: 100%;">
             <tbody>
             <tr>
-                <td><h3>Change Username: </h3></td>
                 <td>
-                    <p>New Username: </p>
-                    {#                        d-flex#}
-                    <form class="d-flex" action="#" method="get">
-                        <input type="text" class="form-control me-2" placeholder=" Enter Username" aria-label="Username"
-                               aria-describedby="basic-addon1" required>
-                        <button class="btn mybtn btn-outline-primary" type="submit">Submit</button>
+                    <form method="post">
+                     {% csrf_token %}
+                        {{ form }}
+                    <button type="submit">Save changes</button>
                     </form>
```

Comment:
@Chris36021 This is how we made it so users can change their password. @gabrieldiazmorro verify that this is correct

Reply:
Esta duro